README.md

```diff
@@ -27,48 +27,29 @@ After installing the above dependencies, install `ng-bootstrap` via:
 ```
 npm install --save @ng-bootstrap/ng-bootstrap
 ```
-Once Installed, directives may be imported in several different ways:
+Once installed Angular 2 modules may be used in two different ways:
+
 For all the directives (if you simply intend on using everything):
 ```
-import {NGB_DIRECTIVES} from '@ng-bootstrap/ng-bootstrap';
-```
-Or, for specific component directives (using Alert as an example):
-```
-import {NGB_ALERT_DIRECTIVES} from '@ng-bootstrap/ng-bootstrap';
-```
-Once imported, add your chosen directives on your component's `directives` array property:
+import {NgbModule} from '@ng-bootstrap/ng-bootstrap';
 ```
-directives: [
-  ...,
-  NGB_DIRECTIVES,
-  ...
-]
+Or, for specific module for one directive (using Alert as an example):
 ```
-Certain directives need to be precompiled in order to be used. This is done via your component's `precompiled` array
-property. The following components require precompilation:
+import {NgbAlertModule} from '@ng-bootstrap/alert';
 ```
-precompile: [
-  ...,
-  NGB_PRECOMPILE,
-  ...
-]
+The only remaining part is to list the imported module in your application module. You should end up with the code similar to:
 ```
-We recommend putting these in the top level component of your application. So ultimately, you
-should wind up having something that looks like this:
-```
-import {Component} from '@angular/core';
-import {NGB_DIRECTIVES, NGB_PRECOMPILE} from '@ng-bootstrap/ng-bootstrap';
-
-@Component({
-  selector: 'app',
-  directives: [ NGB_DIRECTIVES ],
-  precompile: [ NGB_PRECOMPILE ],
-  templateUrl: './app.html',
+import {NgbModule} from '@ng-bootstrap/ng-bootstrap';
+
+@NgModule({
+  declarations: [AppComponent, ..],
+  imports: [NgbModule, ...],  
+  bootstrap: [AppComponent]
 })
-export class App {
-    ...
+export class NgbdModule {
```

Comment:
Isn't it strange to use `NgbdModule` as an example for an application module? Wouldn't `AppModule` be more logical?